pennylane_qiskit/converter.py

```diff
@@ -160,8 +170,24 @@ def _function(params: dict = None, wires: list = None):
             operation_wires = [wire_map[(qubit.register.name, qubit.index)] for qubit in op[1]]
 
             if instruction_name in inv_map and inv_map[instruction_name] in pennylane_ops.ops:
+                # Extract the bound parameters from the operation. If the bound parameters are a
+                # Qiskit ParameterExpression, then replace it with the corresponding PennyLane
+                # variable from the var_ref_map dictionary.
+
+                parameters = []
+                for p in op[0].params:
+
+                    _check_parameter_bound(p, var_ref_map)
```

Comment:
Can't help but feel we can use the return value of `_check_parameter_bound`

Reply:
I've removed the return statement on _check_parameter_bound, since it is never used